### ISSUE TITLE
Let's put some more images and a look to the openjfx.io website

### ISSUE THREAD

**Issue by goxr3plus** [javafxports/openjdk-jfx#396]
When we are talking about UI what impresses users the most is actual screen shots .

Somebody that is new in JavaFX landing in this repository will have no impression at all about what it is .

We need the following on the read me file : 

1) A link to the website : https://openjfx.io/
2) Actual links and screenshots of real projects : 

![chrome_2019-03-04_11-59-35](https://user-images.githubusercontent.com/20374208/53725606-00618e80-3e75-11e9-88e3-5966f59deed4.png)

3) Get started Hello World Links https://openjfx.io/openjfx-docs/

Come on make it more beautiful . I have been writing JavaFX since 2013 and builded big apps .

It's a UI framework ... even Hibernate Git Repo has screenshots ..

:)

**Comment by johanvos** [comment]
That is absolutely true. Can you create an issue (and please also a PR) at https://github.com/openjfx/openjfx.github.io

**Comment by dlemmermann** [comment]
I sent you guys a write up of one of my customer apps (for sports coaches). What happened to that one?

Dirk

**Comment by johanvos** [comment]
can you submit a PR for them? That is much more convenient in making progress on a community website.

**Comment by goxr3plus** [comment]
@johanvos  Hi :) Why should i create thr same issue on the other repository. I was talking about the read me of this repository :)

Let me know :)

**Comment by kevinrushforth** [comment]
@goxr3plus The sort of screen-shots you mention are better left to the [openjfx](https://github.com/openjfx/) project. That project hosts all of the content for openjfx.io.

A link to https://openjfx.io/ from this project seems fine, though.
